test/lib/ansible_test/_internal/commands/integration/cloud/scaleway.py

```diff
@@ -41,13 +41,15 @@ def get_environment_config(self) -> CloudEnvironmentConfig:
 
         env_vars = dict(
             SCW_API_KEY=parser.get('default', 'key'),
-            SCW_ORG=parser.get('default', 'org')
+            SCW_ORG=parser.get('default', 'org'),
+            SCW_PROJECT=parser.get('default', 'project')
```

Comment:
This (and the same usage below) will fail if the user hasn't defined `project`. Can this be changed to use a fallback value for backwards compatibility?